yarn-project/types/src/l2_block.ts

```diff
@@ -153,6 +159,7 @@ export class L2Block {
       newContractData,
       newPublicDataWrites,
       newL1ToL2Messages,
+      newL2ToL1Msgs,
```

Comment:
will this be the hash of the message?

Reply:
If oyu look in the docs [here](https://docs.aztec.network/aztec/how-it-works/l1-l2-messaging#messages) you will see the messages, and then what it is this list is the commitment `sha256(LxLyCrossMsg)`. These is a small update in https://github.com/AztecProtocol/docs/pull/54 for the messages, but it is pending a review from Mike.

Reply:
nice